src/Application/Messenger/Handler/SendRequestConsumer.php

```diff
@@ -56,8 +56,23 @@ public function __invoke(SendRequest $sendRequest): void
             $this->fail($sendRequest->id, "Validation failed: {$this->validator->getReason()}");
         }
 
-        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $sendRequest->params);
-        $bodyPlainText = strip_tags($bodyRenderedHtml);
+        // Instantiate a new Swift Message object
+        $email = new \Swift_Message();
+
+        $images['headerImageRef'] = $this->embedImages($email, 'TBG.png');
+        $images['footerImageRef'] = $this->embedImages($email, 'CCh.png');
+
+        $templateMergeParams = array_merge($images, $sendRequest->params);
+
+        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $templateMergeParams);
+
+        // We conduct a preg replace here because `strip_tags()` doesn't remove css class names between <style></style>
+        // Inspired by: https://stackoverflow.com/a/18089200/6620085
+        $bodyRenderedHtmlWithoutStyles = preg_replace("|<style\b[^>]*>(.*?)</style>|s", "", $bodyRenderedHtml);
```

Comment:
As per other comment, a base template is more flexible. This is also not foolproof, e.g. if we decide to use multiple style tags for the HTML, or other similar tags, it will break.